tests/PHPUnit/Framework/Fixture.php

```diff
@@ -347,6 +350,11 @@ public function clearInMemoryCaches()
         // since Plugin\Manager uses getFromGlobalConfig which doesn't init the config object
     }
 
+    /**
+     * @param \Piwik\Tests\Framework\TestingEnvironmentVariables|null $testEnvironment Ignored.
+     * @param bool|false $testCaseClass Ignored.
+     * @param array $extraPluginsToLoad Ignoerd.
+     */
     public static function loadAllPlugins(TestingEnvironmentVariables $testEnvironment = null, $testCaseClass = false, $extraPluginsToLoad = array())
```

Comment:
I mostly didn't want to break code in plugins. I think ideally, this method should never need to be called... maybe I can deprecate it. Maybe.